scripts/install/bashrc.linux

```diff
@@ -16,5 +16,5 @@ export WEBOTS_ALLOW_MODIFY_INSTALLATION=1                 # If defined, you are
 # These environment variables are necessaries to compile and run Webots #
 #########################################################################
 
-export WEBOTS_HOME=$HOME/webots                                  # Defines the path to Webots home.
-export LD_LIBRARY_PATH=$WEBOTS_HOME/lib/webots:$LD_LIBRARY_PATH  # Add the Webots libraries to the library path (useful when launching Webots directly without using the launcher).
+export WEBOTS_HOME=$HOME/webots                                    # Defines the path to Webots home.
+# export LD_LIBRARY_PATH=$WEBOTS_HOME/lib/webots:$LD_LIBRARY_PATH  # Add the Webots libraries to the library path (useful when launching Webots directly without using the launcher).
```

Comment:
I believe we can even delete this line.
```suggestion
```

Reply:
I followed `JAVA_HOME` and `ROS_DISTRO` examples, but I would also rather delete them.

Reply:
I would say it is good to keep the `JAVA_HOME` and `ROS_DISTRO` commented out as they are needed in case we want to build the corresponding libraries. It is not the case for the `LD_LIBRARY_PATH`.